src/core/polyhedral/libpaths.cc.in

```diff
@@ -0,0 +1 @@
+const char* libcilkPath = "@CILK_PATH@";
```

Comment:
we should not adopt these solutions, rather, let's make change to the cmake and find cilk there and set the CILK_PATH there itself

Reply:
I didn't find any cmake based solution for finding cilk (and I certainly do not want to write one :no_mouth: ).

Reply:
could one not use find_library [which ought look in the default paths]?